examples/hello-world/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "hello-world"
+version = "0.1.0"
+description = "print hello world"
+authors = ["honeycombio"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+honeycomb-beeline = {path = "../../dist/honeycomb_beeline-3.42.99-py3-none-any.whl", develop = false}
```

Comment:
🤩 ooh, is `whl` the magic suffix to get local reference to work?

Reply:
No 😭  I had hoped it would but it works the same as a regular tar.gz file.